sdk/servicebus/azure-servicebus/azure/servicebus/_transport/_pyamqp_transport.py

```diff
@@ -296,7 +296,18 @@ def to_outgoing_amqp_message(annotated_message: "AmqpAnnotatedMessage") -> "Mess
                 reply_to_group_id=annotated_message.properties.reply_to_group_id,
             )
         elif ttl_set:
-            message_properties = Properties(    # type: ignore[call-arg]
+            message_properties = Properties(
+                message_id=None,
+                user_id=None,
+                to=None,
+                subject=None,
+                reply_to=None,
+                correlation_id=None,
+                content_type=None,
+                content_encoding=None,
+                group_id=None,
+                group_sequence=None,
+                reply_to_group_id=None,
```

Comment:
what was the error here that required this change?

Reply:
Missing conditional arguments in call to properties (call-args)